mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/service/impl/MobileAcademyServiceImpl.java

```diff
@@ -183,6 +193,9 @@ public void setBookmark(MaBookmark saveBookmark) {
                 && saveBookmark.getScoresByChapter().size() == CHAPTER_COUNT) {
 
             LOGGER.debug("Found last bookmark and 11 scores. Starting evaluation & notification");
+            // Create an activity record here since pass/fail counts as 1 try
+            activityService.createActivity(
+                    new ActivityRecord(callingNumber.toString(), null, null, null, null, DateTime.now(), ActivityState.COMPLETED));
```

Comment:
I wish java had named parameters so instead of (null, null, null) you could do something like (name=null, somethingElse=null ...)  That or default values.